### PR TITLE
Feature | BACK-91 | Re-add quePosition logic

### PR DESCRIPTION
**Pull request by gcolotti**

### CODE REVIEW

File: src/constants/transactions.js [hunk 1]
```diff
@@ -1,8 +1,10 @@
 export const TRANSACTION = {
   INITIAL_NOTE: 'Welcome to Jibber',
-  EVENT_TYPE: 'NEW_USER',
+  EVENT_TYPE: {
+    NEW_USER: 'NEW_USER'
```
Comment (Benjamindodgson) [inline src/constants/transactions.js:4]: make this INTEREST_PAYMENT

**Review by Benjamindodgson — CHANGES_REQUESTED** [review]
1 change.

File: src/constants/transactions.js (hunk 1 above)
Comment (gcolotti) [reply src/constants/transactions.js:4]: So instead of NEW_USER, make it INTEREST_PAYMENT. Are we doing to use NEW_USER in the future o just remove it?

Comment (gcolotti) [reply src/constants/transactions.js:4]: @Benjamindodgson

Comment (Benjamindodgson) [reply src/constants/transactions.js:4]: just remove it.

**Review by Benjamindodgson — DISMISSED** [review]

**Pull request merged**